docs/configuring-playbook-conduit.md

```diff
@@ -36,7 +36,7 @@ You will have to manually register appservices using the the [register-appservic
 Find the `registration.yaml` in the `/matrix` directory, for example `/matrix/mautrix-signal/bridge/registration.yaml`, then pass the content to Conduit:
 
 
-    @conduit:your.server.name: register-appservice
+    @conduit:example.com: register-appservice
```

Comment:
The change should be fine; `@conduit:matrix.example.com` should be incorrect.

Reply:
Sounds correct